### PR TITLE
Notebook Rendering: HTML tables & KaTeX

### PR DESCRIPTION
**Pull request by Sumukh**
image

### CODE REVIEW

File: server/static/css/notebook.css [hunk 1]
```diff
@@ -15,10 +15,6 @@
     width: inherit !important;
 }
 
-.nb-markdown-cell * {
-    font-family: Helvetica Neue, Helvetica, sans-serif !important;
```
Comment (Sumukh) [inline server/static/css/notebook.css]: This messed with KaTeX rendering of text. I just opted to remove it and allow notebook fonts to match the surrounding page.

**Comment by Sumukh** [comment]
The impact on page load time is small: 
- An extra 85kb on the first load of a code page - which is mostly webfonts for KaTeX from a cdn. 
- Then it's cached and there's no additional impact. 
- KaTeX is pretty performant so it doesn't take that long to render.

**Comment by Sumukh** [comment]
This isn't quite ready to merge yet. 

- `</tbody>` is showing up for some reason 
- KaTeX fonts use relative URLs when they are getting the fonts - which works fine in dev but not in production when bundle everything up.

**Comment by knrafto** [comment]
I hade a similar problem when bundling font-awesome. I tried bundling the fonts too somehow but webassets does not support binary files

**Comment by Sumukh** [comment]
Things should work now. I'm not super happy about having to resort to using `vw` for the max-width of an output - but it works.

**Comment by Sumukh** [comment]
Any comments? This is mostly just some minor CSS tweaks and pulling in some libraries (though it didn't feel like that) 

Probably going to merge this sometime tomorrow if not.

**Pull request merged**